src/data/faq.json

```diff
@@ -1797,7 +1797,8 @@
                         "anchor": "recovered",
                         "active": false,
                         "textblock": [
-                            "Since version 1.13, you have the option to reactivate the app after recovering from a SARS-CoV-2 infection and to restart the risk assessment after sharing your own random IDs. This means that you can now decide if and when to turn risk assessment back on once having shared your encrypted random IDs after a positive test result, thus warning others. Previously, this was only possible via a reset of the app."
+                            "Since version 1.13, you have the option to reactivate the app after recovering from a SARS-CoV-2 infection and to restart the risk assessment after sharing your own random IDs. This means that you can now decide if and when to turn risk assessment back on once having shared your encrypted random IDs after a positive test result, thus warning others. Previously, this was only possible via a reset of the app.",
+                            "In order for the app to show you the 'Risk of Infection' tile again, you need to <b>delete the positive test result from the app</b>. To do this, please tap on the tile that shows the positive test result and tap 'Delete Test' there. After these steps, the test result will be moved to the recycle bin for 30 days and the 'Risk of Infection' tile will be displayed again."

```

Comment:
https://www.coronawarn.app/en/screenshots/#ios_cards calls these "Risk Card", rather than "Risk of Infection" tile.

Reply:
I will change the term, I just used what it's written in the app's "Overview" section.

Reply:
Changed with [`176ce94`](https://github.com/corona-warn-app/cwa-website/pull/2493/commits/176ce9417b360d8740c066d104f29073c5ce1bd8).